app/assets/javascripts/cookie-banner.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable */
```

Comment:
Adding back this file from the old frontend module - temporary fix until VB-366 (new cookie banner) is addressed